data/data/bootstrap/files/usr/local/bin/kubelet-pause-image.sh.template

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -euo pipefail
+# Before kubelet.service and crio.service start, ensure
+# that we're using the pause image from our payload just like the primary cluster.
+# Need to set the --pod-infra-container-image flag for the kubelet to point to the pause image from the payload
+# So we add MACHINE_CONFIG_INFRA_IMAGE to an environment file and source that in the kubelet service
+
+. /usr/local/bin/release-image.sh
```

Comment:
For this to correctly load/finish requires that release image has been downloaded by the release-image.service unit.

How will we ensure that kubelet waits or keeps retrying until I that unit suceeds.
If there is a failure will kubelet service retry or just fail and sit

Reply:
Kubelet can't start until crio has been started. And https://github.com/openshift/installer/blob/master/data/data/bootstrap/systemd/units/crio-configure.service.template#L5 ensures that the release image has been downloaded before crio is started.
But to make sure, I can add `After/Wants=release-image.service` to the kubelet service here https://github.com/openshift/installer/pull/3712/files#diff-367cd6ecdc6c4f5bbe9560468fad15f3R4 as well to ensure you can't start kubelet till release-image.service has been successful. Wdyt?

Reply:
I added `release-image.service` to `After and Wants` in the kubelet service template below.